content/gotraining.slide

```diff
@@ -425,6 +429,53 @@ As part of all labs
 - Create a PR following the [[https://github.com/anz-bank/go-training#pre-pr-checklist][PR checklist]]
 
 
+* PR from own fork - initial setup
+
+Issue PRs from `feature` branch of your own fork to `master` on upstream:
+
+- Login to github.com
+- Click `Fork` in the top right corner of [[https://github.com/anz-bank/go-training-slides]]
+- Set up the following remotes (`USERNAME` is your GitHub username):
+
+	language:none
```

Comment:
curious why this is `language:none` for shell commands, where you use `language:bash` earlier? (and below too)

Reply:
language:bash still does weird things: highlights `clone` and `cd`...

Reply:
nope, only highlights `clone` not cd.
sadly `language:none` has a slightly different color to `language:bash` for default text.
@pentaphobe - how perfectionist are you?

Reply:
reverted it to bash... better have a green `clone` than a different colour default text.